### ISSUE TITLE
Webpack Icon Cut Off in Call Stack Pane

### ISSUE THREAD

**Issue by darkwing** [firefox-devtools/debugger#7993]
Just spotted this:

<img width="330" alt="webpackcallstack" src="https://user-images.githubusercontent.com/46655/53047490-268c3500-3458-11e9-8c5d-60e4944329b5.png">

**Comment by darkwing** [comment]
The Angular icon is also cut off, so I'd assume at this point that all are.

**Comment by chujunlu** [comment]
Hey @darkwing, I was able to reproduce the Angular icon issue and dug a little bit. But could you let me know the steps you did to produce that Webpack icon? Thanks!

**Comment by darkwing** [comment]
I don't recall what React app I tested, but you should be able to reproduce with Angular, but simply change the icon's element className from "angular" to "react".

**Comment by fvsch** [comment]
I think @nchevobbe reported the same issue on Slack and we said that we should remove the `width:12px` from those images and just keep the default 16px size. That should solve it.

**Comment by fvsch** [comment]
@chujunlu, re your questions about why we have some icons defined in `AccessibleImage.css` and some in `SourceIcon.css`:

Basically in #7890 I tried to define `AccessibleImage` as the default, recommended case: monochrome SVG icon used as a mask; and keep `SourceIcon` around as the place where we define icons for file types and libraries. Since library logos are the only use case we had for multicolor icons defined as background-image, I put the `background-*` properties there.

The issue is that the reality of how those components are used is a bit murkier, the same icon with the same semantic intent can be an `<AccessibleImage />` in some place and a `<SourceIcon />` in another place. I think `SourceIcon` is mostly used as a component that does a bunch of work in order to figure out what the library icon should be, and perhaps it shouldn't have any styles of its own, and everything should move to `AccessibleImage.css`.

So I would probably move the `.img.foo` styles from `SourceIcon.css` to `AccessibleImage.css` (perhaps as a second section, rather than sorting everything alphabetically), and the `background-*` styles on `.img` to `AccessibleImage.css` as well.

**Comment by chujunlu** [comment]
Thank you @fvsch . I have a clearer sense now. I missed your discussion about it on slack. Since you've been working on it, I'll be hands-off for this issue. But let me know if I can be of any help.

When I dug around to try to solve the issue, I found that ember and dojo icons have some problem. I happen to try both methods you mentioned: add `background-*` styles on `.img` to `AccessibleImage.css` ; or remove `width: 12px`. Just want to show what I noticed in the screenshot below. Hope it might be helpful for the project. 

<img width="1267" alt="screen shot 2019-02-27 at 3 27 03 pm" src="https://user-images.githubusercontent.com/38041280/53520953-e99def00-3aa4-11e9-83df-2c1c7033f072.png">
